src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -73,6 +75,24 @@ Item {
         id: positionKit
     }
 
+    Rectangle {
+        id: accuracyIndicator
+        visible: withAccuracy
+        x: positionMarker.screenPosition.x - width/2
+        y: positionMarker.screenPosition.y - height/2
+        width: {
```

Comment:
Is this working when positionKit.accuracy changes? Isn't necessary to use [binding](http://doc.qt.io/qt-5/qtqml-syntax-propertybinding.html#using-keyword-this-keyword-with-property-binding)?